<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CoolTestsForm</class>
 <widget class="QDialog" name="CoolTestsForm">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>856</width>
    <height>877</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UsefulClicker Tests Form</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>1330</x>
     <y>30</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>128</width>
     <height>228</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QCheckBox" name="checkBox_2">
      <property name="styleSheet">
       <string notr="true">font: 600 12pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Message 2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Generate</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit_2">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(10, 75, 255);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="plainText">
       <string>Это пример текста который был сгенерирован программой UsefulClicker сегодня.</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>161</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>2 Group. Gmail tests</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton_7">
      <property name="text">
       <string>Compose Message </string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton_6">
      <property name="text">
       <string>Compose Message </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>30</y>
     <width>171</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>1 Group. Notepad tests</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QPushButton" name="typeTest">
      <property name="text">
       <string> Type test</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>Change font </string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="scrollTest">
      <property name="text">
       <string>Scroll test</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>140</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>3 Group. Twitter tests</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton_4">
      <property name="text">
       <string>Post Twit </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>125</width>
     <height>228</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="checkBox">
      <property name="styleSheet">
       <string notr="true">font: 600 12pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Message 1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(10, 75, 255);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="plainText">
       <string>Это пример текста который был сгенерирован программой UsefulClicker сегодня.</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="stopTest">
      <property name="text">
       <string>STOP TEST</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="runFunction">
      <property name="text">
       <string>Run Function</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="functionsList">
      <item>
       <property name="text">
        <string>func1</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>170</y>
     <width>161</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>4 Group. CV tests</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QPushButton" name="buttonsDetector1">
      <property name="text">
       <string>Buttons Detector 1</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QSpinBox" name="spinBox">
      <property name="prefix">
       <string>cannyThreshold = </string>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="imageSearch">
      <property name="text">
       <string>Image Search</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>821</width>
     <height>561</height>
    </rect>
   </property>
   <property name="title">
    <string>Xml editor</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="1" column="0">
     <widget class="QPlainTextEdit" name="xmlEditor">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>11</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPlainTextEdit" name="logEdit">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="plainText">
       <string>x=&quot;&quot; y=&quot;&quot;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>30</y>
     <width>141</width>
     <height>157</height>
    </rect>
   </property>
   <property name="title">
    <string>Pick point from screen</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="screen0">
        <property name="text">
         <string> 0</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="autoRepeat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="screen1">
        <property name="text">
         <string>1</string>
        </property>
        <property name="autoRepeat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="areaButton">
      <property name="text">
       <string>Area selector</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pickPoint">
      <property name="text">
       <string>Coord selector</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="shellDialog">
      <property name="text">
       <string>Shell Dialog</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="buttonsGroup">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>200</y>
     <width>181</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>5. Xml tests</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_7">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>200</y>
     <width>151</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>5 Group. Xml Tests</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_7">
    <item row="0" column="0">
     <widget class="QPushButton" name="createTestXml">
      <property name="text">
       <string>Create Test Document</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="setFunctionTest">
      <property name="text">
       <string>SetFunction Test</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="testStatusLabel">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 900 9pt &quot;Segoe UI&quot;;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(9, 41, 4, 255), stop:0.085 rgba(2, 79, 0, 255), stop:0.19 rgba(50, 147, 22, 255), stop:0.275 rgba(236, 191, 49, 255), stop:0.39 rgba(243, 61, 34, 255), stop:0.555 rgba(135, 81, 60, 255), stop:0.667 rgba(121, 75, 255, 255), stop:0.825 rgba(164, 255, 244, 255), stop:0.885 rgba(104, 222, 71, 255), stop:1 rgba(93, 128, 0, 255));</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;SELECT THE TEST&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CoolTestsForm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CoolTestsForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
